<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModFolderPage</class>
 <widget class="QWidget" name="ModFolderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mods</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="MCModInfoFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="ModListView" name="modTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="rmModBtn">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateModBtn">
           <property name="text">
            <string>&amp;Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="viewModBtn">
           <property name="text">
            <string>&amp;View Folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="orphansBox">
         <property name="toolTip">
          <string>Orphaned mods are mods that where installed as a dependency, but for which the dependee has been uninstalled, and often therefore aren't required anymore</string>
         </property>
         <property name="title">
          <string>Orphaned Mods</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QLabel" name="orphansLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="orphansRemoveBtn">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2" rowspan="3">
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>File System</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="filesystemSearchEdit"/>
           </item>
           <item>
            <widget class="QTreeView" name="filesystemView"/>
           </item>
           <item>
            <widget class="QPushButton" name="filesystemBrowseBtn">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>QuickMods</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="quickmodsSearchEdit"/>
           </item>
           <item>
            <widget class="QListView" name="quickmodsView">
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModListView</class>
   <extends>QTreeView</extends>
   <header>gui/widgets/ModListView.h</header>
  </customwidget>
  <customwidget>
   <class>MCModInfoFrame</class>
   <extends>QFrame</extends>
   <header>gui/widgets/MCModInfoFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
